docs/reference/commandline/ps.md

```diff
@@ -82,20 +82,20 @@ than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bi
 
 The currently supported filters are:
 
-* id (container's id)
+* id (`container's id`)
```

Comment:
Given that the `container's id` is not a literal value, should we swap the formatting for these?

```Markdown
* `id` (container's id)
* `label` (`label=<key>` or `label=<key>=<value>`)
* `name` (container's name)
```

(and so on)